Packs/Akamai_SIEM/Integrations/Akamai_SIEM/Akamai_SIEM.py

```diff
@@ -479,17 +516,26 @@ def main():  # pragma: no cover
             demisto.incidents(incidents)
             demisto.setLastRun(new_last_run)
         elif command == "fetch-events":
-            for events, offset, total_events_count in fetch_events_command(  # noqa: B007
+            page_size = int(params.get("page_size", FETCH_EVENTS_PAGE_SIZE))
+            limit = int(params.get("fetchLimit", 300000))
+            for events, offset, total_events_count, hashed_events_from_previous_run in fetch_events_command(  # noqa: B007
                 client,
-                params.get("fetchTime"),
-                int(params.get("fetchLimit", 20)),
-                params.get("configIds"),
+                "5 minutes",
+                fetch_limit=limit,
+                config_ids=params.get("configIds", ""),
                 ctx=get_integration_context() or {},
```

Comment:
Optional since this is a new thing, but I would add here a small cache that if this isnt the first iteration of the command and we should have the integration context in hand already from the last set, to take it from memory instead of needing to call get_integration_context

Would probably do this by adding wrappers around getintegrationcontext and setintegrationcontext